docs/section12/Rule12-3.md

```diff
@@ -1,38 +1,58 @@
-  - For each zone in thermal block: ```zone_proposed in thermal_block_proposed.zones:```
-  - For each space in thermal zone: ```space_proposed in zone_proposed.spaces:```  
-    - Get the ID of the space: ```space_ID_proposed = space_proposed.ID```
-    - Add to the list of space IDs in the Proposed model: ```space_IDs_proposed_list.append(space_ID_proposed)```
 
-**Rule Assertion:** ```sorted(space_IDs_user_list) == sorted(space_IDs_proposed_list)```  
-
-**[Back](../_toc.md)**
+## Rule Logic:  
+- Iterate through the spaces in the proposed model: `for space_p in P_RMD...spaces`  
+  - Get the lighting space type: `space_type_p = space_p.lighting_space_type` 
+  - Iterate through the miscellaneous equipment loads in the space: `for misc_equip_p in space_p.miscellaneous_equipment:`  
+    - Get the proposed automatic receptacle control: `auto_receptacle_control_p = misc_equip_p.has_automatic_control`  
+    - If the space type is not in the list of space types where receptacle controls may be required, and the space has automatic receptacle controls installed: `if space_type_p not in EXPECTED_RECEPTACLE_CONTROL_SPACE_TYPES and auto_receptacle_control_p:`  
+      - Get the percentage of controlled receptacles: `controlled_percentage = misc_equip_p.automatic_controlled_percentage`
+      - Calculate the expected receptacle power credit: `expected_receptacle_power_credit = 0.10 * controlled_percentage`
+      - Get the associated miscellaneous equipment object in the baseline model: `misc_equip_b = get_component_by_id(misc_equip_p.id, B_RMD)`
+      - Get the baseline miscellaneous equipment multiplier schedule: `hourly_multiplier_schedule_b = misc_equip_b.multiplier_schedule`
+      - Get the proposed miscellaneous equipment multiplier schedule: `hourly_multiplier_schedule_p = misc_equip_p.multiplier_schedule`
+      - Create a list of expected hourly values based on the expected receptacle power credit: `expected_hourly_values = [hour_value * (1 - expected_receptacle_power_credit) for hour_value in hourly_multiplier_schedule_b.hourly_values]`
+      - If it is a leap year, set the schedule comparison mask to 8784 hours, else set it to 8760 hours: `mask_schedule = [1] * 8784 if is_leap_year else [1] * 8760`
+      - Compare the expected miscellaneous equipment load schedule to the proposed miscellaneous equipment load schedule: `credit_comparison_data = compare_schedules(expected_hourly_values, hourly_multiplier_schedule_p, mask_schedule)`
+      - Compare the baseline miscellaneous equipment load schedule to the proposed miscellaneous equipment load schedule: `no_credit_comparison_data = compare_schedules(hourly_multiplier_schedule_b, hourly_multiplier_schedule_p, mask_schedule)`
+
+
+**Rule Assertion:**  
+- Case 1: If the expected receptacle power credit was applied as expected for all hours: PASS `if credit_comparison_data['total_hours_matched'] == len(hourly_multiplier_schedule_p.hourly_values == len(expected_hourly_values): PASS` 
```

Comment:
Shouldn't this be indented? This logic is for each applicable misc obj in each space. Having it out here makes it apply globally.